import-export-cli/utils/constants.go

```diff
@@ -112,5 +112,14 @@ const DefaultTokenType = "JWT"
 
 var ValidInitialStates = []string{"CREATED", "PUBLISHED"}
 
+var EnvReplaceFilePaths  = []string{
```

Comment:
Shall we change the small formatting issue here? Should be a space after "[] string"